setup.py

```diff
@@ -18,8 +18,8 @@ def read_readme():
 INSTALL_REQUIRES = [
     "boto3 >= 1.14.20",
     "botocore >= 1.17.20",
-    "google-api-python-client >= 1.7.7, < 2.49.0",
-    "protobuf < 3.12.0",
+    "google-api-python-client >= 1.7.7",
```

Comment:
I think the `< 2.49.0` was added because anything greater than that is 3.7+ only.

Reply:
@TheRealFalcon I can do that.

Reply:
> @holmanb , can you test this version out in the env that was giving you failure?

@TheRealFalcon: Test result here:
https://app.travis-ci.com/github/canonical/cloud-init/jobs/572383639

Reply:
@holmanb the protobuf versioning was wrong, let's see if CI passes now.

Reply:
[Rerunning now](https://app.travis-ci.com/github/canonical/cloud-init/jobs/572387109)

Reply:
Looks like it passed!